rcloud.support/R/rcloud.support.R

```diff
@@ -495,8 +495,14 @@ rcloud.is.notebook.visible <- function(id)
     visibility
   }
 
-rcloud.set.notebook.visibility <- function(id, value)
+rcloud.set.notebook.visibility <- function(id, value){
   rcloud.set.notebook.property(id, "visible", value != 0);
+  if(value){
+    response <- update.solr(rcloud.get.notebook(id),rcloud.notebook.star.count(id))
```

Comment:
b5c282922a3325b59e633c45c5f01a0a4ab6eb76 should fix the R type 24 to JSON error in binary assets